lib/src/upgrade_alert.dart

```diff
@@ -117,11 +117,7 @@ class UpgradeAlertState extends State<UpgradeAlert> {
             }
 
             if (!displayed) {
-              final checkContext = widget.navigatorKey != null &&
-                      widget.navigatorKey!.currentContext != null
-                  ? widget.navigatorKey!.currentContext!
-                  : context;
-              checkVersion(context: checkContext);
+              checkVersion(context: context);
```

Comment:
@RyanCarrier By not using `widget.navigatorKey!.currentContext` in the function `checkVersion`, it is not used by downstream functions, such as `widget.upgrader.determineMessages`, which eventually is used to determine the locale. I don't fully understand the implications of this change.